repository/iris_venues_queries.py

```diff
@@ -0,0 +1,11 @@
+from typing import List
+
+from models.db import db
+
+
+def link_user_to_nearest_iris(latitude: float, longitude: float) -> List:
```

Comment:
```suggestion
def link_user_to_iris(latitude: float, longitude: float) -> List:
```
ce nom de fonction ne correspond pas exactement à ce que fait la fonction :
avec 'nearest' on comprend que même si l'utilisateur n'était pas dans un iris il en aurait quand même un d'associé,
on te propose ce nom si ça te parait cohérent

Reply:
Yes done !